trieste/models/gpflow/models.py

```diff
@@ -211,6 +232,183 @@ def evaluate_loss_of_model_parameters() -> tf.Tensor:
 
         multiple_assign(self.model, current_best_parameters)
 
+    def conditional_predict_f(
+        self, query_points: TensorType, additional_data: Dataset
+    ) -> tuple[TensorType, TensorType]:
+        """
+        Returns the marginal GP distribution at query_points conditioned on both the model
+        and some additional data, using exact formula.
```

Comment:
perhaps a pointer where these formulas can be found?

Reply:
we use shape abbreviations here again, we should be clear somewhere what they stand for

Reply:
so this is from https://arxiv.org/pdf/1203.6452.pdf
eq 8 to 10, right?

Reply:
correct, thx - will update the docstring